style.css

```diff
@@ -120,21 +163,25 @@ main {
 }
 
 button {
-  background-color: #0026ff;
+  background: #584d42a6;
   color: white;
   border: none;
   border-radius: 4px;
   padding: 16px 40px;
   margin-right: 4px;
+  margin-top: 5%;
   font-size: 16px;
   line-height: 26px;
   font-family: 'Montserrat';
   font-weight: 500;
   cursor: pointer;
   transition: all 0.3s ease;
```

Comment:
We only see the transition effect when hovering over the button, so what does this transition do to the button (when not hovering)?

Reply:
we think is ease in the effect when you hovering over the item. Slow down the change to the hovering state